.gitignore

```diff
@@ -0,0 +1,125 @@
+i# Byte-compiled / optimized / DLL files
```

Comment:
Is the leading `i` here a typo?

Reply:
You are right - it is indeed a typo. I have fixed that in https://github.com/shagunsodhani/stochastic_gradient_push/commit/9d971e78b7e65ba729d74fb367ebfc33d9dfe3b2